pantheon/src/main/java/tech/pegasys/pantheon/controller/IbftPantheonController.java

```diff
@@ -286,6 +286,12 @@
           } catch (final InterruptedException e) {
             LOG.error("Failed to shutdown ibft processor executor");
           }
+          timerExecutor.shutdownNow();
+          try {
+            timerExecutor.awaitTermination(5, TimeUnit.SECONDS);
+          } catch (final InterruptedException e) {
+            LOG.error("Failed to shutdown timerexecutor");
```

Comment:
nit: thinking this should be "Failed to shutdown timer executor"

Reply:
done.